site/en/docs/lighthouse/performance/bootup-time/index.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Reduce JavaScript execution time
+description: |
+  Learn how JavaScript execution can slow down your page performance
+  and how you can speed it up.
+date: 2019-05-02
+updated: 2019-10-04
+web_lighthouse:
```

Comment:
Is this `web_lighthouse` frontmatter metadata used anywhere, or is it just historical? I took a quick look through the d.c.c. templates, and I don't see where it's being used.

Reply:
No idea! I kept it for now because it's certainly not breaking anything, but I can try removing it to see if it busts anything and then pull it out of all the posts that use it.

Reply:
👍 

I'm sensitive to this because dealing with how 11ty merges metadata in posts is apparently the biggest source of performance issues for our current web.dev build (https://github.com/GoogleChrome/web.dev/issues/7339), and I am concern about creeping build times for d.c.c. as well.